src/app/CreateRecording/CreateRecording.tsx

```diff
@@ -65,9 +66,9 @@ const Comp: React.FunctionComponent< RouteComponentProps<{}, StaticContext, Crea
 
   const [activeTab, setActiveTab] = React.useState(0);
 
-  const handleCreateRecording = (recordingName: string, events: string, duration?: number): void => {
+  const handleCreateRecording = (recordingAttributes: RecordingAttributes): void => {
     addSubscription(
-      context.api.createRecording({ recordingName, events, duration })
+      context.api.createRecording({ recordingAttributes })
```

Comment:
`func({ key })` is syntactic sugar for `func({ key: key })` - that is, the function is being called with an object as its sole parameter, and that object has a single key. This can/should be simplified.